consensus/src/marshal/ingress/handler.rs

```diff
@@ -88,3 +94,395 @@ impl<K: Array> Producer for Handler<K> {
         receiver
     }
 }
+
+/// A request for backfilling data.
+#[derive(Clone)]
+pub enum Request<B: Block> {
+    Block(B::Commitment),
+    Finalized { height: u64 },
+    Notarized { view: u64 },
+}
+
+impl<B: Block> Request<B> {
+    /// The predicate to use when pruning subjects related to this subject.
+    ///
+    /// Specifically, any subjects unrelated will be left unmodified. Any related
+    /// subjects will be pruned if they are "less than" this subject.
```

Comment:
"less than" -> "less-than-or-equal-to" ?

Reply:
Good catch